mmdet3d/core/bbox/structures/utils.py

```diff
@@ -175,10 +175,10 @@ def mono_cam_box2vis(cam_box):
     assert isinstance(cam_box, CameraInstance3DBoxes), \
         'input bbox should be CameraInstance3DBoxes!'
 
-    loc = cam_box.gravity_center
-    dim = cam_box.dims
-    yaw = cam_box.yaw
-    feats = cam_box.tensor[:, 7:]
+    loc = cam_box.gravity_center.clone()
```

Comment:
This function will be deprecated soon and we won't use it in the future. Your fix are correct but I'd suggest you not using this function in your code :)